### PR TITLE
Bail if we're in customizer

### PR DESCRIPTION
**Pull request by aidvu**
Users were seeing errors when in Customizer. Just skip
all concat/minify stuff when in there.

### CODE REVIEW

File: concat-js.php [hunk 1]
```diff
@@ -217,7 +217,7 @@ function do_items( $handles = false, $group = false ) {
 						return $path;
 					}, $js_array['paths'] );
 					$mtime = max( array_map( 'filemtime', $paths ) );
-					$path_str = implode( ',', $js_array['paths'] ) . "?m=${mtime}j";
```
Comment (aidvu) [inline concat-js.php:220]: This is horrible. :laughing:

File: page-optimize.php [hunk 2]
```diff
@@ -132,9 +132,18 @@ function page_optimize_sanitize_exclude_field( $value ) {
...
+	require_once __DIR__ . '/settings.php';
+	require_once __DIR__ . '/concat-css.php';
+	require_once __DIR__ . '/concat-js.php';
 
-// Disable Jetpack photon-cdn for static JS/CSS
-add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
+	// Disable Jetpack photon-cdn for static JS/CSS
+	add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
+}
+add_action( 'plugins_loaded', 'page_optimize_init' );
```
Comment (aidvu) [inline page-optimize.php:149]: It's kind-of late loading our plugin, but haven't seen any performance differences between this and loading as we did before.

The thing with the old way is... I can't detect customizer (or haven't figured out a way to).

File: concat-js.php (hunk 1 above)
Comment (brandonpayton) [reply concat-js.php:220]: I've seen that happen mistakenly with trying to use vim movement while in insert mode... thankfully, this one already existed in the code we built upon. 😄

**Comment by brandonpayton** [comment]
I think we should just be able to avoid this issue with an `is_admin()` check in `concat-css.php`. We already have one in `concat-js.php` to prevent us from changing admin JS. I'd assumed we already had that for CSS.

**Comment by aidvu** [comment]
I believe I tried that and got JS concat errors in the Customizer, as it didn't hit the `admin` check. Will test tomorrow. If it's enough, we can move the check in the loader, instead of separate files.

**Pull request merged**